packages/server/src/lib/ab.ts

```diff
@@ -66,31 +61,40 @@ export const selectVariant = <V extends Variant, T extends Test<V>>(test: T, mvt
 };
 
 export const selectAmountsTestVariant = (
-    test: ModifiedChoiceCardAmounts,
+    tests: AmountsTests,
+    countryCode: string,
     countryGroupId: CountryGroupId,
     mvtId: number,
 ): SelectedAmountsVariant | undefined => {
-    const regionTest = test[countryGroupId];
+    // Two-tier amounts - check for country test, then region test
+    let testArray = tests.filter(t => t.target === countryCode);
```

Comment:
I think you mentioned this offline, but what about using `find` here since we only need 1 test?

Reply:
Will recode as suggested

Reply:
Done (needs testing)